tools/clangrttool/clangrttool.py

```diff
@@ -21,10 +21,17 @@
 """
 
 import os
+import re
 import subprocess
 import sys
 import zipfile
 
+def normalize_clang_lip_path(path, local_developer_dir):
+  return re.sub(".*/Contents/Developer",
```

Comment:
```suggestion
  return re.sub(".*\.app/Contents/Developer",
```

marginally safer, no need to take it if you don't want to

Reply:
I'll take it.
And yes, ready to go.